seqr/views/utils/orm_to_json_utils.py

```diff
@@ -696,7 +696,7 @@ def get_json_for_gene_notes_by_gene_id(gene_ids, user):
     return notes_by_gene_id
 
 
-def get_json_for_locus_lists(locus_lists, user, include_genes=False, include_project_count=False, is_analyst=None):
+def get_json_for_locus_lists(locus_lists, user, include_genes=False, include_pagenes=False, include_project_count=False, is_analyst=None):
```

Comment:
Is there a reason to make `include_pagenes` its own kwarg? What would the case be where `include_genes` is True and `include_pagenes` is False?

Reply:
Yes, when first querying for all genelists (`/summary_data/gene_lists`), `include_genes` is set to true so UI filter can include filtering by gene symbol.  However, the filter in the UI do not require any of the fields from `pagene` so including all of them upfront increases the returned json payload size a fair bit and slows things down.